netzob/src/netzob/Model/Vocabulary/Symbol.py

```diff
@@ -169,6 +169,41 @@ def specialize(self, memory=None, generationStrategy=None, presets=None):
         :rtype: :class:`str``
         :raises: :class:`netzob.Model.Vocabulary.AbstractField.GenerationException` if an error occurs while generating a message
         """
+
+        if presets is not None:
```

Comment:
Difficult to understand. Would it be possible to include some doctests to illustrate how presets can be use ?